core/dbt/utils.py

```diff
@@ -514,6 +514,8 @@ def translate_aliases(kwargs, aliases):
 def pluralize(count, string):
     if count == 1:
         return "{} {}".format(count, string)
+    elif string[-2:] == 'is':   # Latin third declension i-stem
```

Comment:
can you just check if `string` == `analysis` here? I like latin as much as the next guy, but the string fiddling here _non est necesse_